### PR TITLE
Add Instagram to the Social Media (SM)

### PR DESCRIPTION
**Pull request by jilusan**

### CODE REVIEW

**Comment by jilusan** [comment]
@shawnthompson 
Could you help for this one?
Thanks.

**Comment by MRicher** [comment]
@jilusan Your fork is behind the original, just fetch the original and correct the conflict.

**Comment by jilusan** [comment]
Yes. my fort (and me) are behind... The problem is to do it correctly.

**Comment by LaurentGoderre** [comment]
```bash
git fetch upstream
git rebase upsream/master
```

**Comment by jilusan** [comment]
Yes. I did. 
I got:
On branch master
Your branch is ahead of 'origin/master' by 63 commits.

**Comment by LaurentGoderre** [comment]
git rebase should end up in an error due to conflict

**Comment by MaximMartel** [comment]
@nschonni, he didnt changed tabs to spaces in _base.scss.

File: src/social-media-icons/_base.scss [hunk 1]
```diff
@@ -2,127 +2,203 @@
...
+%social-media-icons-followus-logo-properties {
+    background-position: center center;
```
Comment (nschonni) [inline src/social-media-icons/_base.scss]: @MaximMartel this is spaces, previous line 6 is tabs as set out in the EditConfig and SCSSLint settings

**Comment by MaximMartel** [comment]
@nschonni no more white space, correct?

**Comment by MaximMartel** [comment]
I am ready to merge this PR, please dismiss your review if the issue with white space is no longer present @nschonni

**Pull request merged**